src/cdomains/apron/apronDomain.apron.ml

```diff
@@ -64,7 +64,7 @@ struct
   type mt = Box.t
   type t = mt Manager.t
   let mgr = Box.manager_alloc ()
-  let name = "Interval Manager"
+  let name () = "Interval"
 end
 
 let priv_manager =
```

Comment:
I guess this is a leftover from #418, but it shouldn't be called "priv" anymore.